apps/web/src/components/ui/healthcare/healthcare-loading.tsx

```
@@ -0,0 +1,566 @@
+              <p className='text-xs text-red-600 dark:text-red-400'>
+                {details}
+              </p>
+            )}
+          </div>
+          {onDismiss && (
+            <button
+              onClick={onDismiss}
+              className='flex-shrink-0 text-red-500 hover:text-red-700 dark:text-red-400 dark:hover:text-red-300 focus:outline-none focus:ring-2 focus:ring-red-500 rounded-sm p-1'
+              aria-label='Fechar mensagem de erro'
+            >
+              <svg className='h-4 w-4' fill='none' viewBox='0 0 24 24' stroke='currentColor'>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)